templates/tensorflow-gpu-1.12/image_classifier/src/example.py

```diff
@@ -0,0 +1,57 @@
+import Algorithmia
```

Comment:
i think this file should be named __ALGORITHM__.py (which will get autofilled in with the name of the algorithm).  the pipe script is going to try loading a file with that name, and otherwise an algorithm made with this template won't have a main entrypoint

Reply:
oh ok I wasn't sure what the pipeline looked like, I'll do that

Reply:
i misspoke earlier it's just `__ALGO__.py` https://github.com/algorithmiaio/langpacks/tree/master/python/template/src